src/convertToCase/getErrors.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const getErrors = (currText, toCase) => {
+  const payload = [];
+  const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  if (!currText.length) {
```

Comment:
you can use just (!currText) instead of (!currText.length)

Reply:
agree)